packages/foam-vscode/package.json

```diff
@@ -15,14 +15,52 @@
   ],
   "activationEvents": [
     "workspaceContains:.vscode/foam.json",
-    "onCommand:foam-vscode.update-wikilinks"
+    "onCommand:foam-vscode.update-wikilinks",
+    "onCommand:foam-vscode.open-daily-note"
   ],
   "main": "./out/extension.js",
   "contributes": {
     "commands": [
       {
         "command": "foam-vscode.update-wikilinks",
         "title": "Foam: Update Markdown Reference List"
+      },
+      {
+        "command": "foam-vscode.open-daily-note",
+        "title": "Foam: Open Daily Note"
+      }
+    ],
+    "configuration": {
+      "title": "Foam",
+      "properties": {
+        "foam.openDailyNote.fileExtension": {
+          "type": "string",
+          "scope": "resource",
+          "default": "md"
+        },
+        "foam.openDailyNote.filenameFormat" : {
+					"type": "string",
```

Comment:
The indentation here seems a little excessive. Can we use spaces over tabs?

Easiest way to fix this is to run Prettier on the file (and generally, set it to format on save)